pgcli/pgclirc

```diff
@@ -22,14 +22,11 @@ multi_line = False
 # a command.
 multi_line_mode = psql
 
-# Destructive warning mode will alert you before executing a sql statement
+# Destructive warning will alert you before executing a sql statement
 # that may cause harm to the database such as "drop table", "drop database",
-# "shutdown", "delete", or "update".
-# Possible values:
-# "all" - warn on data definition statements, server actions such as SHUTDOWN, DELETE or UPDATE
-# "moderate" - skip warning on UPDATE statements, except for unconditional updates
-# "off" - skip all warnings
-destructive_warning = all
+# "shutdown", "delete", or "update". You can pass a list of destructive 
+# commands or "off" if you want to skip all warnings.
```

Comment:
Let's not tell people to use `off`, this is an "old style" value now. They can leave the setting empty. I.e `destructive_warning =`.

Reply:
Sounds good. The reason I thought initially about keeping `off` was for the `--warn` option that overrides the config. But I guess one could always pass `--warn off` since it's backwards compatible.